### DISCUSSION TITLE
【question】证明内容里的普通数字为斜体

### DISCUSSION THREAD

**Discussion by fantasyEthan** [nju-lug/NJUThesis#132] [General]
<img width="461" alt="image" src="https://user-images.githubusercontent.com/54493995/163961658-190b9813-9f64-4abf-829e-288ce0ef2f46.png">
<img width="193" alt="image" src="https://user-images.githubusercontent.com/54493995/163961708-56b0ca7d-b27b-4a63-800c-a69f361733ad.png">
应该怎么更改为正常的非斜体？

**Comment by atxy-blip** [comment]
在这里简单提供两种方法：

1. `性质$1$`，即使用数学模式下的数字
2. `性质~{\normalfont\ref{label}}`，即在引用标签时套一个指定字体的命令

第二点可以定义出成一个自己使用的命令，比如

```TeX
\newcommand\refproperty[1]{性质~{\normalfont\ref{#1}}}
```

------

乍一看以为是需要全盘改整个 `proof` 环境内的字体，打算重开 #50 细化一下设置项。

**Reply by fantasyEthan** [reply]
谢谢作者的解答，我也知道可以通过局部修改的方式达到目的；这个问题算是一种enhancement（辛苦作者了哈哈哈？或者说有统一的接口去修改，因为不仅是引用的数字，在证明环境下普通的数字也会出现斜体（暂时先用数学环境替代了）

**Reply by atxy-blip** [reply]
理论上可以[通过 Unicode 字符范围](https://stackoverflow.com/questions/23738423/using-unicode-range-to-exclude-numbers-only)来单独指定数字的字体样式，但我才疏学浅，目前折腾不到这步 Orz
